docs-new/docs/graphql/core/api-reference/metadata-api/source.mdx

```diff
@@ -101,19 +100,18 @@ X-Hasura-Role: admin
 }
 ```
 
-### Args syntax
+### Args syntax {#metadata-pg-drop-source-syntax}
 
 | Key     | Required | Schema                    | Description                                                                                                                                                        |
 |---------|----------|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| name    | true     | `SourceName <SourceName>` | Name of the Postgres database                                                                                                                                      |
+| name    | true     | [SourceName](/graphql/core/api-reference/syntax-defs.mdx#sourcename) | Name of the Postgres database                                                                                                                                      |
 | cascade | false    | Boolean                   | When set to `true`, the effect (if possible) is cascaded to any metadata dependent objects (relationships, permissions etc.) from other sources (default: `false`) |
 
-## rename_source
+## rename_source [#metadata-rename-source]
```

Comment:
broken custom id syntax - might be a typo, please fix
```suggestion
## rename_source {#metadata-rename-source}
```

Reply:
Done